portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFileEntryLocalServiceImpl.java

```diff
@@ -243,12 +243,15 @@ public DLFileEntry addFileEntry(
 
 		String actualExtension = null;
 
-		if(inputStream instanceof ByteArrayFileInputStream){
-			actualExtension = FileUtil.getExtension(((ByteArrayFileInputStream)inputStream).getFile().getName());
+		if (inputStream instanceof ByteArrayFileInputStream) {
+			actualExtension = FileUtil.getExtension(
+				((ByteArrayFileInputStream)inputStream).getFile(
+				).getName());
```

Comment:
There's no way this will go through Brian. Can you please make the code follow the conventions? Also, try to rebase the pull request so that commits are as clean as possible and pass SF on its own. Otherwise, things are difficult to review.